src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -983,6 +983,9 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
     <value>Could not update the client certificate: {0}</value>
     <comment>0 - reason</comment>
   </data>
+  <data name="Error_CouldNotUpdateMetadataLastAccessTime" xml:space="preserve">
+    <value>Updating last access time on file {0} failed with {1}</value>
```

Comment:
Please add a comment describing the 0 and 1 to help the localization folks. 

Somehing like 0 - file path, 1 - failure reason

Reply:
Done.

Reply:
Perfect.